### DISCUSSION TITLE
Module access - deny by default option?

### DISCUSSION THREAD

**Discussion by ghferrari** [spring-projects/spring-modulith#923] [General]
First of all, thanks to the Spring Modulith team for this great package :-)

If I've understood the documentation for version 1.2.5, Spring Modulith currently uses an "allow-by-default" access model i.e. access to other modules is allowed unless [explicitly restricted](https://docs.spring.io/spring-modulith/reference/fundamentals.html#modules.explicit-dependencies).

Is there a way to switch to a "deny-by-default" permission model? In this scenario, access to other modules would be forbidden by default unless specifically granted?

I think this would be a useful feature.

**Comment by odrotbohm** [comment]
That's a neat suggestion. I've created GH-994 to track the idea officially.
